aten/src/ATen/native/quantized/cpu/qlinear.cpp

```diff
@@ -900,6 +901,96 @@ at::Tensor PackedLinearWeightsOnednn:: apply_tanh(
       std::move(input), output_scale, output_zero_point);
 }
 
+static at::Tensor linear_int8_with_onednn_weight(
+    at::Tensor input, // int8 CPU Tensor, not QTensor
+    double input_scale,
+    int64_t input_zero_point,
+    at::Tensor mkldnn_weight, // int8 tensor from MkldnnCPU
```

Comment:
can we use consistent names, i.e., `onednn_weight`?

Reply:
Thanks. It's fixed.